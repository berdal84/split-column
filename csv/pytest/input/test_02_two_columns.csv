title,version
Alien;Terminator,1;2;3
Avatar;Harry Potter,1;2