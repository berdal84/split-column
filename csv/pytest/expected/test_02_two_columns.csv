title,version
Alien,1
Alien,2
Alien,3
Terminator,1
Terminator,2
Terminator,3
Avatar,1
Avatar,2
Harry Potter,1
Harry Potter,2
